xml/snmp_sysdescr.xml

```diff
@@ -660,6 +660,18 @@
     <param pos="1" name="os.product"/>
   </fingerprint>
 
+  <!--======================================================================
+                              ALLWORX
+   =======================================================================-->
+
+  <fingerprint pattern="^ALLWORX (9\d{3}.*)$">
```

Comment:
The pattern will match any characters that follow a 9 and 3 digits. For example, the value `9111XYZ` is matched and captured. Is this intentional? If so, please include a valid example for this match.

Reply:
I believe this was intentional, but I don't have a specific example that goes beyond the 9XXX match (the specific example we had is included a few lines below: "ALLWORX 9308"). I believe this match was to "keep the window open" for matches of other ALLWORX 9XXX devices as found on their website (http://www.allworx.com/ip-phones), as some models end in G, L, Ex, though I know that I did not go looking in shodan or other places to verify they appear in the same fashion. I'll see if I can find a match and add it here, also happy to adjust this pattern if preferred.

Reply:
A [quick shodan search](https://www.shodan.io/search?query=port%3A161+%22allworx+93%2A%22&page=1) didn't reveal any devices with alpha chars at the end, I'll trim the search to just look for 9XXX.